test-suite/CMakeLists.txt

```diff
@@ -77,11 +77,31 @@ add_djinni_target(DjinniWCharTests
   YAML_OUT_FILE     WCHAR_YAML_GENERATED_SRCS
 )
 
+add_djinni_target(DjinniPyTests
+  IDL "djinni/py_all.djinni"
+  PY_OUT "${CMAKE_CURRENT_BINARY_DIR}/generated-src/python"
+  PYCFFI_PACKAGE_NAME "PyCFFIlib"
+  PYCFFI_DYNAMIC_LIB_LIST "mylib"
+  PYCFFI_OUT "${CMAKE_CURRENT_BINARY_DIR}/generated-src/cffi"
+  C_WRAPPER_OUT "${CMAKE_CURRENT_BINARY_DIR}/generated-src/cwrapper"
+  CPP_OUT "${CMAKE_CURRENT_BINARY_DIR}/generated-src/pycpp"
+  CPP_NAMESPACE "testsuite"
+  IDENT_CPP_ENUM_TYPE "foo_bar"
+  CPP_OPTIONAL_TEMPLATE "std::experimental::optional"
+  CPP_OPTIONAL_HEADER "\"test-suite/handwritten-src/cpp/optional.hpp\""
+
+  PY_OUT_FILES         PYTHON_GENERATED_SRCS
+  PYCFFI_OUT_FILES     PYCFFI_GENERATED_SRCS
+  C_WRAPPER_OUT_FILES  C_WRAPPER_GENERATED_SRCS
+  CPP_OUT_FILES        PYCPP_GENERATED_SRCS
+)
 
 file(GLOB_RECURSE CPP_HANDWRITTEN_SRCS "handwritten-src/cpp/*")
 file(GLOB_RECURSE JAVA_TEST_SRCS "handwritten-src/java/*")
 file(GLOB_RECURSE OBJC_HANDWRITTEN_SRCS "handwritten-src/objc/impl/*")
 file(GLOB_RECURSE OBJC_TEST_SRCS "handwritten-src/objc/tests/*")
+file(GLOB_RECURSE C_WRAPPER_HANDWRITTEN_SRCS "handwritten-src/cwrapper/*")
```

Comment:
I'd like to see the tests use the targets defined in the root `CMakeLists.txt`, instead of globbing for the lib sources. But we do not have to do this now, I've already openend an issue for that: #22